### PR TITLE
Add support for older Geist/ITWatchdog relaygoose

### PR DESCRIPTION
**Pull request by Moonlight63**
Add support for older Geist/ITWatchdog relaygoose

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [ ] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by Moonlight63** [comment]
I don't know how to make it recheck the CLA. I signed it, but it didn't work.

**Comment by murrant** [comment]
@Moonlight63 sorry, the CLA script doesn't work if your github email and commit email don't match.

**Comment by murrant** [comment]
Also, your sensors should be defined in yaml if possible.

**Comment by Moonlight63** [comment]
Oh, OK, I guess I entered a different email? My bad. As for the sensors, I just copied the format that already existed for the sensors that matched the closest. I did define the discovery in yaml if that's what you are talking about. Otherwise I don't exactly know what I'm doing, I just needed it to work, and this was the only way it seemed to work. If I got rid of the php files then none of the sensors were being scanned.

File: includes/discovery/sensors/airflow/wxgoos.inc.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ *
+ * @copyright  2023 Dalen Catt
+ * @author     Dalen Catt <dalencattmlsp@gmail.com>
+ */
+$value = snmp_get($device, 'climateAirflow', '-Oqv', 'IT-WATCHDOGS-MIB-V3');
+$current_oid = '.1.3.6.1.4.1.17373.3.2.1.9.1';
+$descr = 'Airflow';
+if (is_numeric($value)) {
+    discover_sensor($valid['sensor'], 'airflow', $device, $current_oid, 'climateAirflow', 'wxgoos', $descr, 1, 1, null, null, null, null, $value);
+}
```
Comment (PipoCanaja) [inline includes/discovery/sensors/airflow/wxgoos.inc.php:31]: Hi @Moonlight63 
This one can be converted into YAML.

File: includes/discovery/sensors/temperature/wxgoos.inc.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<?php
```
Comment (PipoCanaja) [inline includes/discovery/sensors/temperature/wxgoos.inc.php:1]: This file can also easily migrate to YAML

File: includes/polling/sensors/pre-cache/wxgoos.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<?php
```
Comment (PipoCanaja) [inline includes/polling/sensors/pre-cache/wxgoos.php:1]: Same

File: includes/polling/sensors/temperature/wxgoos.php [hunk 4]
```diff
@@ -0,0 +1,28 @@
+<?php
```
Comment (PipoCanaja) [inline includes/polling/sensors/temperature/wxgoos.php:1]: Same

**Review by PipoCanaja — CHANGES_REQUESTED** [review]

**Comment by PipoCanaja** [comment]
Hi @Moonlight63 
As this PR seems to be stalled, we'll close it for now. Whenever you have time to continue it, feel free to re-open.
Thanx

**Pull request closed without merging**